### PR TITLE
Make OE re-percieve aromaticity in `to_openeye`

### PR DESCRIPTION
**Pull request by j-wags**
Quick fix for #644 

The quick fix simply has Molecule.to_openeye disregard the OFFMol's existing is_aromatic flags and re-perceives them. In almost all cases, this should be fine.

For more details, see: https://github.com/openforcefield/openforcefield/issues/644#issuecomment-658422943

### CODE REVIEW

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -91,7 +91,7 @@ jobs:
         conda activate test
         if [[ "$RDKIT" == true ]]; then conda install rdkit; fi
         if [[ "$OPENEYE" == true ]]; then
-          conda install openeye-toolkits=2019
+          conda install openeye-toolkits
```
Comment (mattwthompson) [inline .github/workflows/CI.yml:94]: I could see the case for `openeye-toolkikts >=2020` ... probably better to leave it off - how much do we worry about users having old versions of the toolkit installed? If they _did_ (and they shouldn't) then the errors may be cryptic.

File: openforcefield/utils/toolkits.py [hunk 2]
```diff
@@ -1331,7 +1331,7 @@ def to_openeye(molecule, aromaticity_model=DEFAULT_AROMATICITY_MODEL):
         for atom in molecule.atoms:
             oeatom = oemol.NewAtom(atom.atomic_number)
             oeatom.SetFormalCharge(atom.formal_charge.value_in_unit(unit.elementary_charge)) # simtk.unit.Quantity(1, unit.elementary_charge)
-            oeatom.SetAromatic(atom.is_aromatic)
+            #oeatom.SetAromatic(atom.is_aromatic)
```
Comment (mattwthompson) [inline openforcefield/utils/toolkits.py:1334]: Please add a note here if we intend to un-comment it in the future, or (better, I think) remove this altogether since it does not fit into the medium- or long-term plans you proposed

File: openforcefield/utils/toolkits.py [hunk 3]
```diff
@@ -1347,12 +1347,14 @@ def to_openeye(molecule, aromaticity_model=DEFAULT_AROMATICITY_MODEL):
             oebond = oemol.NewBond(oemol_atoms[atom1_index],
                                    oemol_atoms[atom2_index])
             oebond.SetOrder(bond.bond_order)
-            oebond.SetAromatic(bond.is_aromatic)
+            #oebond.SetAromatic(bond.is_aromatic)
```
Comment (mattwthompson) [inline openforcefield/utils/toolkits.py:1350]: Same as above

File: openforcefield/tests/test_molecule.py [hunk 4]
```diff
@@ -708,9 +708,9 @@ def test_to_from_iupac(self, molecule):
                               'DrugBank_3726', 'DrugBank_3844', 'DrugBank_3930', 'DrugBank_4161',
                               'DrugBank_4162', 'DrugBank_4778', 'DrugBank_4593', 'DrugBank_4959',
                               'DrugBank_5043', 'DrugBank_5076', 'DrugBank_5176', 'DrugBank_5418',
-                              'DrugBank_5737', 'DrugBank_5902', 'DrugBank_6304', 'DrugBank_6305',
-                              'DrugBank_6329', 'DrugBank_6355', 'DrugBank_6401', 'DrugBank_6509',
-                              'DrugBank_6531', 'DrugBank_6647',
+                              'DrugBank_5737', 'DrugBank_5902', 'DrugBank_6295', 'DrugBank_6304',
```
Comment (mattwthompson) [inline openforcefield/tests/test_molecule.py:711]: A bigger issue to tackle than in here, but is there documentation about why these lists of molecules are problematic and why?

File: .github/workflows/CI.yml (hunk 1 above)
Comment (j-wags) [reply .github/workflows/CI.yml:94]: This should actually be reverse-compatible! So there's no need to pin OE or worry about older versions :-)

File: openforcefield/utils/toolkits.py (hunk 3 above)
Comment (j-wags) [reply openforcefield/utils/toolkits.py:1360]: Added a TODO item to revisit the relationship between OFFMols' and OEMols' aromatic flags in a96274f

File: openforcefield/utils/toolkits.py (hunk 2 above)
Comment (j-wags) [reply openforcefield/utils/toolkits.py]: Added a TODO item to revisit the relationship between OFFMols' and OEMols' aromatic flags in a96274f

File: openforcefield/tests/test_molecule.py (hunk 4 above)
Comment (j-wags) [reply openforcefield/tests/test_molecule.py:711]: No. This block is a reservoir of technical debt. Dealing with edge cases in OE and RDK is kinda like running a series of seismographs. When they have big releases it's just like "oh, a bunch of stuff behaves differently now", but to actually go and check+fix every road and power line along the fault would be a full-time job in itself. 

Even worse, I'm sure some of the cases that previously violated one of our assumptions are being _fixed_ in some of these updates. But to catch those we'd need a whole matrix of true/fale negatives/positives. So, it's something that a really cheminformatics-inclined person could set up in the future. But we try to avoid the rabbit hole of doing cheminformatics here.

Comment (davidlmobley) [reply openforcefield/tests/test_molecule.py:711]: I just made a comment in a related issue about aromaticity models, which is probably the reason behind SOME of these.

Comment (davidlmobley) [reply openforcefield/tests/test_molecule.py:711]: That was over here, FTR: https://github.com/openforcefield/openforcefield/issues/644#issuecomment-666605131
